volume-providers/vpc/vpcclient/client/client_test.go

```diff
@@ -193,7 +194,7 @@ func TestDebugMode(t *testing.T) {
 			operation: getOperation,
 			verify: func(t *testing.T) {
 				assert.Contains(t, log.String(), "REQUEST:")
-				assert.Contains(t, log.String(), "GET /resource?version="+models.APIVersion+" HTTP/1.1")
+				assert.Contains(t, log.String(), "GET /resource?version=2019-01-01 HTTP/1.1")
```

Comment:
Should this be `2019-07-02`

Reply:
This is just a test case and values are evaluated on this basis, so its fine for unit tests